pkgs/development/python-modules/pypcap/default.nix

```diff
@@ -1,34 +1,63 @@
-{ lib, writeText, buildPythonPackage, fetchPypi, libpcap, dpkt }:
+{ lib
+, buildPythonPackage
+, dpkt
+, fetchFromGitHub
+, fetchpatch
+, libpcap
+, pytestCheckHook
+, writeText
+}:
 
 buildPythonPackage rec {
   pname = "pypcap";
   version = "1.2.3";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1w5i79gh7cswvznr8rhilcmzhnh2y5c4jwh2qrfnpx05zqigm1xd";
+
+  src = fetchFromGitHub {
+    owner = "pynetwork";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1zscfk10jpqwxgc8d84y8bffiwr92qrg2b24afhjwiyr352l67cf";
   };
 
   patches = [
     # The default setup.py searchs for pcap.h in a static list of default
     # folders. So we have to add the path to libpcap in the nix-store.
     (writeText "libpcap-path.patch"
```

Comment:
Can we move this inline patch to a file?

Reply:
I replaced it with `substituteInPlace`. It's a very specific patch which could not be re-used elsewhere.